### PR TITLE
Added social links

### PR DESCRIPTION
**Pull request by samyakbvs**
google, github, and flickr
#235 fixed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -953,6 +953,9 @@ <h1>How Can I Join Projects</h1>
 								<li><a target="_self" href="https://www.linkedin.com/groups/FOSSASIA-3762811"><i class="icon social_linkedin icon-large"></i></a></li>
 								<li><a target="_self" href="https://www.youtube.com/channel/UCQprMsG-raCIMlBudm20iLQ"><i class="icon social_youtube icon-large"></i></a></li>
 								<li><a target="_self" href="https://www.flickr.com/photos/fossasia"><i class="icon social_instagram icon-large"></i></a></li>
+								<li><a target="_self" href="https://github.com/fossasia"><i class="icon social_github icon-large"></i></a></li>
```
Comment (akshatnitd) [inline index.html]: The github social icon doesn't exist. Try testing it.

File: index.html [hunk 2]
```diff
@@ -953,6 +953,9 @@ <h1>How Can I Join Projects</h1>
...
+								<li><a target="_self" href="https://plus.google.com/108920596016838318216"><i class="icon social_google icon-large"></i></a></li>
```
Comment (akshatnitd) [inline index.html]: The google social icon doesn't exist. Try testing it.

Replace both icons with valid one.

**Comment by samyakbvs** [comment]
Done @akshatnitd

File: index.html [hunk 3]
```diff
@@ -36,6 +36,7 @@
      ga('create', 'UA-17146009-5', 'auto');
      ga('send', 'pageview');
     </script>
+	<script defer src="https://use.fontawesome.com/releases/v5.0.1/js/all.js"></script>
```
Comment (abishekvashok) [inline index.html]: pls use themify icons

**Comment by samyakbvs** [comment]
Please see @akshatnitd and @abishekvashok

**Comment by akshatnitd** [comment]
@samyakbvs Please send a demo link

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/samyak-labs.fossasia.org/

**Comment by samyakbvs** [comment]
@akshatnitd

**Comment by mariobehling** [comment]
Please also add the Github icon in the top menu bar on the left same as fossasia.org.

**Comment by samyakbvs** [comment]
sure

**Comment by samyakbvs** [comment]
@mariobehling

**Comment by ParthS007** [comment]
@samyakbvs  update your preview link and PR is ready to merge 👍

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/samyakjain-bvs-labs.fossasia.org/

**Comment by samyakbvs** [comment]
👍

**Pull request merged**